server/src/main/java/org/elasticsearch/persistent/PersistentTasksClusterService.java

```diff
@@ -45,12 +48,14 @@
 
     private final ClusterService clusterService;
     private final PersistentTasksExecutorRegistry registry;
+    private final AssignmentDecider<PersistentTaskParams> decider;
```

Comment:
can we just hard code this for now to EnableAssignmentDecider? we can generalize when it becomes relevant (and know what we want exactly)